modules/tiles/test/tileset/tileset-3d.spec.js

```diff
@@ -543,9 +546,16 @@ test('Tileset3D#handles failed tile processing', t => {
 test('Tileset3D#loads tiles in tileset', async (t) => {
   const tilesetJson = await load(TILESET_URL, Tiles3DLoader);
   const tileset = new Tileset3D(tilesetJson);
+  // @ts-ignore
   tileset.root._visible = true;
-  await tileset.root.loadContent();
-  t.ok(tileset.root.content);
+
+  if (tileset.root) {
```

Comment:
Is it OK to have a tileset that doesn't have a root? Perhaps it would be good to require `root` not to be optional?

Reply:
If we take a look into `tileset-3d.ts` file `this.root `can be null sometimes.